src/main/java/io/appium/java_client/android/AndroidDriver.java

```diff
@@ -176,4 +179,16 @@ public void openNotifications() {
     public void toggleLocationServices() {
         CommandExecutionHelper.execute(this, toggleLocationServicesCommand());
     }
+
+    public void toggleWifi() {
+        CommandExecutionHelper.execute(this, toggleWifiCommand());
+    }
+
+    public void toggleAirplaneMode() {
```

Comment:
docstrings are missing. BTW, this method works for Android only. Also, it foes not work since Android 7+

Reply:
In such case, what to do ?

Reply:
apply the interface to Android driver only. Also, describe the limitations in the docstring

Reply:
This works only on emulator, should we move it inside SupportsSpecialEmulatorCommands

Reply:
> should we move it inside SupportsSpecialEmulatorCommands
not needed. As you said Appium will anyway throw an exception if the command didn't work. Although, it might be useful to describe what is supported in the docstring